### ISSUE TITLE
pyrobuf resources not found when compiling proto file

### ISSUE THREAD

**Issue by claws** [appnexus/pyrobuf#39]
I am getting errors when trying to compile the [metrics.proto](https://github.com/prometheus/client_model/blob/master/metrics.proto) file from the Prometheus client model project. It appears that some of the referenced ``pyrobuf`` internal packages can not be found during the processing of the proto file.

I am running in a virtual environment (to isolate it from other work) which may be causing the issue. Is there any reason that pyrobuf should not work from within a virtual environment?

To create the environment I am running in I ran the following steps:
```
$ git clone https://github.com/appnexus/pyrobuf.git
$ pyvenv pyrobuf_venv
$ cd pyrobuf_venv
$ source bin/activate
$ cd ..
$ cd pyrobuf
$ python setup.py install
```

I then tested my setup using:
```
$ python
Python 3.5.1 (v3.5.1:37a07cee5969, Dec  5 2015, 21:12:44) 
[GCC 4.2.1 (Apple Inc. build 5666) (dot 3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import pyrobuf
>>> import pyrobuf_list
>>> 
```

Then I tried to compile my proto file which produces a lot of errors like these:

```
$ python -m pyrobuf metrics.proto 
generating metrics.proto
missing cimport in module 'pyrobuf_list': out/metrics_proto.pyx
missing cimport in module 'pyrobuf_list': out/metrics_proto.pxd
Compiling out/metrics_proto.pyx because it changed.
[1/1] Cythonizing out/metrics_proto.pyx

Error compiling Cython file:
------------------------------------------------------------
...
from libc.stdint cimport *
from libc.string cimport *
from cpython.ref cimport PyObject

from pyrobuf_list cimport *
^
------------------------------------------------------------

out/metrics_proto.pxd:5:0: 'pyrobuf_list.pxd' not found

Error compiling Cython file:
------------------------------------------------------------
...
    cdef bint _sample_count_isSet
    
    cdef double _sample_sum
    cdef bint _sample_sum_isSet
    
    cdef TypedList _quantile
        ^
------------------------------------------------------------

out/metrics_proto.pxd:148:9: 'TypedList' is not a type identifier

Error compiling Cython file:
------------------------------------------------------------
...
    pass

cdef class LabelPair:

    def __cinit__(self):
        self._listener = <PyObject *>null_listener
                                                 ^
------------------------------------------------------------

out/metrics_proto.pyx:17:50: undeclared name not builtin: null_listener
```

The pyrobuf files seem to be installed.
```
$ ls /path/to/my/venv/lib/python3.5/site-packages/pyrobuf-0.5.8-py3.5-macosx-10.6-intel.egg/
EGG-INFO/                           pyrobuf/                            pyrobuf_list.py                     pyrobuf_util.py
__pycache__/                        pyrobuf_list.cpython-35m-darwin.so  pyrobuf_util.cpython-35m-darwin.so 
```

**Comment by claws** [comment]
So I got out of my virtual environment and installed ``pyrobuf`` (0.5.8) directly into my Python3.5 and tried to use ``pyrobuf`` to process the ``metrics.proto`` file. It worked. This makes me think that the issue is related to running within a virtual environment. Some kind of path related issue for the compiler from within the virtual env I assume.

**Comment by claws** [comment]
This is a real problem in continuous integration environments such as Travis-CI, which typically run in virtual environments. I'm guessing it is something to do with the include paths being supplied to ``cythonize``.

**Comment by claws** [comment]
I think I've identified the problem. The MANIFEST.in is missing the following entries:
```
include pyrobuf/src/pyrobuf_list.pxd
include pyrobuf/src/pyrobuf_list.pyx
```
which results in these files not being copied in site-packages when ``pyrobuf`` is installed. Then later when I attempt to use ``pyrobuf`` to build my project artefacts I get the error (snippet):
```
from pyrobuf_list cimport *
^
out/metrics_proto.pxd:5:0: 'pyrobuf_list.pxd' not found
```

With the existing MANIFEST.in file, the ``site-packages/pyrobuf-0.5.10-py3.5-linux-x86_64.egg/pyrobuf/src/`` directory contains:
```
ansidecl.h  pyrobuf_list.c  pyrobuf_util.c  pyrobuf_util.pxd  pyrobuf_util.pyx  strndup.c
```

With the MANIFEST.in file modified to include the missing files the installed ``site-packages/pyrobuf-0.5.10-py3.5-linux-x86_64.egg/pyrobuf/src/`` directory contains:
```
ansidecl.h  pyrobuf_list.c  pyrobuf_list.pxd  pyrobuf_list.pyx  pyrobuf_util.c  pyrobuf_util.pxd  pyrobuf_util.pyx  strndup.c
```

Looking at the history for the ``MANIFEST.in`` file it appears that it used copy all files: 
```
include pyrobuf/src/*.pyd
include pyrobuf/src/*.pyx
```
but was recently modified to only explicitly copy the utils file. Perhaps it needs to go back to how it was.


I think I was incorrect in one of my earlier posts where I said that the problem seemed to disappear when I exited the virtual environment. I think in that case I just reverted to an install executed using ``pip install -e .`` which simply links back to the dev area rather than doing a real install. This would explain why the ``pyrobuf_list.*`` files were able to be resolved.
